templates/scripts/run_dm-master.sh.tpl

```diff
@@ -17,9 +17,9 @@ cd "${DEPLOY_DIR}" || exit 1
 {{- end}}
 
 {{- if .NumaNode}}
-exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/dm-master \
+exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/bin/dm-master \
```

Comment:
It's better to use `bin/dm-master` instead of `bin/bin/dm-master`

Reply:
https://github.com/pingcap/tiup/issues/557
add ` avoid use bin/bin/ in the deploy dir` in the issue, will not do this in this pr.
we always install the component in the {deploy_dir}/bin in the `InstallPackage` task.